site/en/guide/data_performance.ipynb

```diff
@@ -0,0 +1,766 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Tce3stUlHN0L"
+   },
+   "source": [
+    "##### Copyright 2019 The TensorFlow Authors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
Use "Omit code cell output when saving this notebook". I think this should get all the execution counts down to 0.

Reply:
This is a Colab feature, I use Jupyter. I cleared all cells output before saving, this should put `execution_count` to `null`, is it OK?